app/controllers/katello/api/v2/host_subscriptions_controller.rb

```diff
@@ -0,0 +1,81 @@
+module Katello
+  class Api::V2::HostSubscriptionsController < Katello::Api::V2::ApiController
+    respond_to :json
```

Comment:
This isn't necessarily required -- https://github.com/Katello/katello/blob/master/app/controllers/katello/concerns/api/api_controller.rb#L9